Server/htdocs/AppController/commands_RSM/itemsManager/classLbxUsers_updateUser.php

```diff
@@ -3,62 +3,86 @@
 require_once "../utilities/RSdatabase.php";
 require_once "../utilities/RSMitemsManagement.php";
 require_once "../utilities/RSMdefinitions.php";
+require_once "../utilities/RSMbadgesManagement.php";
 
+isset($GLOBALS['RS_POST']['password']) ? $password = $GLOBALS['RS_POST']['password'] : $password = "";
+isset($GLOBALS['RS_POST']['badge'   ]) ? $badge    = $GLOBALS['RS_POST']['badge'   ] : $badge = RSgetUniqueBadge($GLOBALS['RS_POST']['clientID']);
+$login    = base64_decode($GLOBALS['RS_POST']['login']);
 $clientID = $GLOBALS['RS_POST']['clientID'];
-$userID = $GLOBALS['RS_POST']['userID'];
-$login = base64_decode($GLOBALS['RS_POST']['login']);
-$password = $GLOBALS['RS_POST']['password'];
+$userID   = $GLOBALS['RS_POST']['userID'  ];
 $personID = $GLOBALS['RS_POST']['personID'];
 
-//First of all, we need to check if the variable clientID does not have the value 0
+
+// Check that the received clientID is a valid number
 if ($clientID > 0) {
-	//We check if the user already exists
+	
+	// We check if the user already exists
 	$theQuery_userExists = "SELECT RS_USER_ID, RS_PASSWORD FROM rs_users WHERE RS_USER_ID = ".$userID." AND RS_CLIENT_ID = ".$clientID;
 	$resultUsers = RSQuery($theQuery_userExists);
+	
 	if ($resultUsers->fetch_array() != 0) {
 			// The user exists, so check if the login is valid
```

Comment:
Add more clarity to the comment: we are preventing to update a user with an email that already matches another user in the same customer